sample-app/app/src/main/java/com/socure/idplus/MainActivity.kt

```diff
@@ -119,11 +105,10 @@ class MainActivity : AppCompatActivity(), MultiplePermissionsListener,
         list.add(DeviceRiskManager.DeviceRiskDataSourcesEnum.Advertising)
         //Locale
         list.add(DeviceRiskManager.DeviceRiskDataSourcesEnum.Locale)
-        //Bluetooth
-        //list.add(DeviceRiskManager.DeviceRiskDataSourcesEnum.Bluetooth)
 
         list.add(DeviceRiskManager.DeviceRiskDataSourcesEnum.Network)
-        list.add(DeviceRiskManager.DeviceRiskDataSourcesEnum.Exif)
+        //Accessibility
+        list.add(DeviceRiskManager.DeviceRiskDataSourcesEnum.Accessibility)
 
         sharedPref = getSharedPreferences(PREF_NAME, PRIVATE_MODE)
         uuid = sharedPref?.getString(getString(R.string.uuidKey), null)
```

Comment:
Please remove this line and the one above. Also remove initialization of sharedPref in line 33

Reply:
Also line 116